arm-siterecovery/2015-11-10/swagger/service.json

```diff
@@ -0,0 +1,13707 @@
+      "get": {
+        "tags": [
+          "VCenters"
+        ],
+        "summary": "Gets the details of a vCenter.",
+        "description": "TODO (avrai): Detailed Description.",
+        "operationId": "VCenters_Get",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "#/parameters/ResourceName"
+          },
+          {
+            "$ref": "#/parameters/ResourceType"
+          },
+          {
+            "$ref": "#/parameters/ResourceNamespace"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionId"
+          },
+          {
+            "name": "fabricName",
+            "in": "path",
+            "description": "The name of the fabric containing the vCenter.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "vCenterName",
+            "in": "path",
+            "description": "vCenter name of interest.",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/VCenter"
+            }
+          }
+        },
+        "deprecated": false
+      },
+      "put": {
+        "tags": [
+          "VCenters"
+        ],
+        "summary": "Perform Add vCenter operation.",
+        "description": "TODO (avrai): Detailed Description.",
+        "operationId": "VCenters_Create",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "#/parameters/ResourceName"
+          },
+          {
+            "$ref": "#/parameters/ResourceType"
+          },
+          {
+            "$ref": "#/parameters/ResourceNamespace"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionId"
+          },
+          {
+            "name": "fabricName",
+            "in": "path",
+            "description": "The name of the fabric containing the vCenter.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "vCenterName",
+            "in": "path",
+            "description": "The name of the vCenter.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "addVCenterRequest",
+            "in": "body",
```

Comment:
is there a reason for the body parameter not being the same model as the response?
 M2017: The model definition for the body parameter and the response MUST be the same for a PUT operation. This ensures that the same entity will be reusable between GET and PUT. (from https://github.com/Azure/azure-rest-api-specs/blob/62c96876795e600de8369efe42edd3a70f494017/documentation/swagger-checklist.md)